togetherly/lib/models/chore.dart

```diff
@@ -8,6 +8,8 @@ enum ChoreStatus {
 }
 
 class Chore {
+  final int id; //TODO set to auto increment in the database
+  final Person assignedPerson;
```

Comment:
Since we want to be able to support assigning multiple children to a chore, this should probably be a list. Also, to avoid having to load everything recursively, relationships like this should be represented as IDs in the model classes. So I'm thinking this should be a list of person/child IDs.